### PR TITLE
Expose `PROPERTY_USAGE_READ_ONLY`

### PR DESCRIPTION
**Pull request by Zylann**
This is needed when registering read-only properties in extensions or `_get_property_list`.

### CODE REVIEW

File: core/core_constants.cpp [hunk 1]
```diff
@@ -651,6 +651,7 @@ void register_global_constants() {
 	BIND_CORE_ENUM_CONSTANT(PROPERTY_USAGE_DEFAULT);
 	BIND_CORE_ENUM_CONSTANT(PROPERTY_USAGE_DEFAULT_INTL);
 	BIND_CORE_ENUM_CONSTANT(PROPERTY_USAGE_NO_EDITOR);
+	BIND_CORE_ENUM_CONSTANT(PROPERTY_USAGE_READ_ONLY);
```
Comment (akien-mga) [inline core/core_constants.cpp:654]: This should be bound in the same order it's declared in the enum (as the binding order is kept in the docs).

**Comment by YuriSizov** [comment]
You also need to re-run the doctool after changing the binding order :)

File: doc/classes/@GlobalScope.xml [hunk 2]
```diff
@@ -2818,6 +2818,9 @@
 		</constant>
 		<constant name="PROPERTY_USAGE_EDITOR_BASIC_SETTING" value="134217728" enum="PropertyUsageFlags">
 		</constant>
+		<constant name="PROPERTY_USAGE_READ_ONLY" value="268435456" enum="PropertyUsageFlags">
+			The property is read-only.
```
Comment (rburing) [inline doc/classes/@GlobalScope.xml:2822]:
```suggestion
			The property is read-only in the editor inspector.
```

The flag only affects the inspector.

Comment (Zylann) [reply doc/classes/@GlobalScope.xml]: Updated.

**Review by rburing — APPROVED** [review]

**Review by akien-mga — APPROVED** [review]

**Comment by akien-mga** [comment]
Thanks!

**Pull request merged**